istioctl/docker/Dockerfile.istioctl

```diff
@@ -0,0 +1,20 @@
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
+
+# Version is the base image version from the TLD Makefile
+ARG BASE_VERSION=latest
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM docker.io/istio/base:${BASE_VERSION} as default
+
+USER 1337:1337
```

Comment:
Why 1337? I thought this was only needed in proxy images for iptables to function correctly.

Reply:
I stole this from galley, originally. The actual value of 1337 is not critical here. I just assumed that was the standard uid used by istio components.

Changed to 1000 to avoid implying that this image has any sort of privilege.